src/Services/Search/ParametersHandler.php

```diff
@@ -68,8 +68,9 @@ public function handlePaginationAndSorting($search, $request)
         $isFiltersSet = array_key_exists('attrib', $request->all());
 
         $defaultSort = $isSearch ? $config->sorting->defaultSortingSearch : $config->sorting->defaultSorting;
-
-        if (!in_array($defaultSort, Plugin::API_SORT_ORDER_AVAILABLE_OPTIONS)) {
+        if (!$isSearch && !$isFiltersSet) {
```

Comment:
When `!$isSearch` the defaultSort is already set to `$config->sorting->defaultSorting;`, so is this relevant to check again?